docs/DatabaseCatalogAPI.yaml

```diff
@@ -0,0 +1,530 @@
+      description: Get metadata about this API.
+      tags:
+        - "metadata"
+      security:
+        - ApiKeyAuth: []
+      responses:
+        200:
+          description: Successful pull of the metadata.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Metadata"
+
+
+components:
+  schemas:
+    GtfsFeed:
+      type: object
+      properties:
+        id:
+          type: string
+          example: feed_0
+        data_type:
+          type: string
+          enum:
+            - gtfs
+          example: gtfs
+        features:
+          type: array
+          items:
+            type: string
+            example: fares-v2
+        status:
+          $ref: "#/components/schemas/FeedStatus"
+        location:
+          $ref: "#/components/schemas/FeedLocation"
+        provider:
+          type: string
+          example: London Transit Commission
+        name:
+          type: string
+          example: feed_0
+        urls:
+          $ref: "#/components/schemas/FeedUrls"
+
+    GtfsFeeds:
+      type: array
+      items:
+        $ref: "#/components/schemas/GtfsFeed"
+
+
+    FeedId:
+      type: object
+      properties:
+        feed_id:
+          type: string
+          example: feed_0
+      required:
+        - feed_id
+
+    FeedName:
+      type: string
+      example: Feed 0
+
+    FeedUrls:
+      type: object
+      properties:
+        direct_download:
+          type: string
+          format: url
+          example: http://www.londontransit.ca/gtfsfeed/google_transit.zip
+        authentication_type:
+          type: integer
+          enum:
+            - 0
+            - 1
+            - 2
+        authentication_info_url:
+          type: string
+          format: url
+        api_key_parameter_name:
+          type: string
+          description: The api_key_parameter_name field defines the name of the parameter to pass in the URL to provide the API key. This field is required for authentication_type=1 and authentication_type=2.
+        latest:
+          type: string
+          format: url
+          example: https://storage.googleapis.com/storage/v1/b/mdb-latest/o/us-maine-casco-bay-lines-gtfs-1.zip?alt=media
+
+    FeedLocation:
+      type: object
+      properties:
+        country_code:
+          description: Should be an enum from https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes
+          type: string
+          example: USA
+        subdivision_name:
+          type: string
+          example: California
+        municipality:
+          type: string
+          example: Los Angeles
+        bounding_box:
+          $ref: "#/components/schemas/DatasetGeographicalData"
+
+    FeedStatus:
+      type: string
+      enum:
+        - active
+        - deprecated
+        - inactive
+        - development
+      example: inactive
+
+    Feeds:
+      type: array
+      items:
+        $ref: "#/components/schemas/Feed"
```

Comment:
Removed

Reply:
As you suggested in the past we should have homogenous data in the response. We could start with only returning a list of feed IDs, names and types. 
It would be useful to determine the data that is common to all feeds and put this in the response also.

Reply:
Great! I love starting with a simple solution and building on it 🚀.

Reply:
Added a feeds endpoint that returns all types of feeds. The returned data incudes a bit more than IDs, names and types. We could remove some and add some. TDB